target/linux/ramips/dts/rt5350_olimex_rt5350f-olinuxino-evb-plus.dts

```diff
@@ -0,0 +1,119 @@
+/dts-v1/;
+
+#include "rt5350_olimex_rt5350f-olinuxino-evb.dts"
+
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "olimex,rt5350f-olinuxino-evb-plus", "ralink,rt5350-soc";
+	model = "Olimex RT5350F-OLinuXino-EVB+";
+
+	aliases {
+		spi0 = &spi_mmc;
+		spi1 = &spi_uext;
+	};
+
+	gpio_export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+
+		acin {
+			gpio-export,name = "acin";
+			gpios = <&gpio0 21 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	spi_mmc: spi-mmc {
+		compatible = "spi-gpio";
+		#address-cells = <1>;
+		ranges;
+
+		gpio-sck = <&gpio0 4 GPIO_ACTIVE_HIGH>;
+		gpio-miso = <&gpio0 6 GPIO_ACTIVE_HIGH>;
+		gpio-mosi = <&gpio0 5 GPIO_ACTIVE_HIGH>;
+		cs-gpios = <&gpio0 3 GPIO_ACTIVE_HIGH>, <&gpio1 5 GPIO_ACTIVE_HIGH>;
+		num-chipselects = <2>;
+
+		m25p80@0 {
+			compatible = "jedec,spi-nor";
+			reg = <0>;
+			spi-max-frequency = <10000000>;
```

Comment:
Higher frequency possible?

Reply:
Actually I've made a mistake. When using spi-gpio, the frequency cannot be changed. This can be seen here:
https://elixir.bootlin.com/linux/v4.14.158/source/drivers/spi/spi-gpio.c#L134

I've measured the maximum possible frequency: 500000. I'm going to fix this.